tests/storage/outofprocess_test.py

```diff
@@ -31,6 +31,7 @@
 import weakref
```

Comment:
Regarding the commit message - it mostly describes how the code works, but we can see this in the code.

What we cannot see in the code is:
- What is the issue with pytest tmpdir?
- What is the advantage of using userstorage?

The commit message should answer these questions.

The commit message can add some details about the implementation, if the details
are important to the change. I'm not sure the details about how the file is created are important.

The blank line between Fixes: and Signed-off-by: should be removed. We should treat "Fixes:" as a header to make it easier for process git logs with tools.